eng/pipelines/tagger.yml

```diff
@@ -93,7 +64,26 @@ steps:
     Set-Location '$(Pipeline.Workspace)/VS'
     git init
```

Comment:
I don't think the working directory is needed here, so `git init --bare` should be faster. It will only create the `.git` folder, essentially.

Reply:
With checkout not required, using bare is an option now. I tested them locally but given we are initializing new repo, default is master branch which is empty and we don't do checkout. I didn't see any difference in terms. The data transferred and size of folder were same. I will add bare just to make sure we don't cause checkout in future accidentally